driver-embedded/src/main/com/mongodb/embedded/client/MongoEmbeddedSettings.java

```diff
@@ -30,6 +30,49 @@
 @Immutable
 public final class MongoEmbeddedSettings {
     private final String libraryPath;
+    private final String yamlConfig;
+    private final LogLevel logLevel;
+    private final String applicationData;
+
+    /**
+     * LogLevel represents the supported logging levels for the embedded mongod
+     */
+    public enum LogLevel {
+        /**
+         * Turn off logging
+         */
+        NONE(0),
+
+        /**
+         * Log to stdout
+         */
+        STDOUT(1),
+
+        /**
+         * Log to stderr
+         */
+        STDERR(2),
```

Comment:
This is not yet implemented in the server but added for future completeness as using it doesn't cause an error.

Reply:
Ack